pkgs/servers/computing/slurm/default.nix

```diff
@@ -8,11 +8,16 @@
 
 stdenv.mkDerivation rec {
   name = "slurm-${version}";
-  version = "17.11.5";
+  version = "17.11.7";
 
```

Comment:
The current `slurm` build doesn't work if you try to rebuild from source, since the upstream tarball has disappeared after schedmd swapped it to the latest release.